wm_2dnav/config/base_local_planner_params.yaml

```diff
@@ -33,10 +33,15 @@ TrajectoryPlannerROS:
   #The weighting for how much the controller should stay close to the path it was given. default 0.6
   pdist_scale: 0.3
   # The weighting for how much the controller should attempt to avoid obstacles. default 0.01
-  occdist_scale: 0.7
+  occdist_scale: 0.2
```

Comment:
Are those change related to the new repo structure or have been introduced by mistake?